modules/cli/src/main/kotlin/datamaintain/cli/App.kt

```diff
@@ -47,10 +46,10 @@ class App : CliktCommand() {
             props.load(it.inputStream())
         }
         overloadPropsFromArgs(props)
-        props.put("dbType", dbType)
     }
 
     private fun overloadPropsFromArgs(props: Properties) {
+        dbType?.let { props[CoreConfigKey.DB_TYPE.key] = it }
```

Comment:
If you see on my PR I also do an other check and put to be backward compatible.

Reply:
Yes very good idea, I added backward compatibility!